src/odemis/driver/actuator.py

```diff
@@ -2697,11 +2699,15 @@ def adjustFocus(self, future, vector_value, rel):
         :param rel: whether it's a relative movement or absolute
         """
         # Drop focus adjustment if lens is parked
-        if self._isParked():
-            logging.warning("Focus adjust movement is dropped as lens is parked.")
+        # if self._isParked():
+        #     logging.warning("Focus adjust movement is dropped as lens is parked.")
+        #     return
+        if not self._isInRange():
+            logging.warning("Focus adjust movement is dropped as lens is not in active range.")
             return
         if rel:
-            self._doMoveRel(future, {'z': vector_value})
+            # Move the underlying lens with the relative shift value
+            execute_function_within_subfuture(future, self._lensz.moveRel, {'z': vector_value})
         else:
             self._doMoveAbs(future, vector_value, adjust=True)
```

Comment:
Please, do the same with absolute moves. It should be "symmetric". This way, you can drop the "adjust" argument of doMoveAbs() and the function will be much simpler.